ecs-logs-collector.sh

```diff
@@ -797,6 +806,80 @@ enable_ecs_agent_debug() {
   fi
 }
 
+disable_docker_debug() {
+  try "disable debug mode for the Docker daemon"
+
+  if [ -e /etc/sysconfig/docker ] && ! grep -q "^\\s*OPTIONS=\"-D" /etc/sysconfig/docker; then
+    info "Debug mode is already disabled."
+  else
+
+    if [ -e /etc/sysconfig/docker ]; then
+      case "${init_type}" in
+        systemd)
+          # sed -i 's/^OPTIONS="\(.*\)/OPTIONS="-D \1/g' /etc/sysconfig/docker
+          sed -i 's/\-D//g' /etc/sysconfig/docker
+          ok
+
+          try "restart Docker daemon to enable debug mode"
+          systemctl restart docker.service
+          ok
+          ;;
+        *)
+          # echo "OPTIONS=\"-D \$OPTIONS\"" >> /etc/sysconfig/docker
+          sed -i 's/\-D//g' /etc/sysconfig/docker
+
+          try "restart Docker daemon to enable debug mode"
+          service docker restart
+          ok
+
+        esac
+
+    else
+      warning "the current operating system is not supported."
+    fi
+  fi
+}
+
+disable_ecs_agent_debug() {
+  try "disable debug mode for the Amazon ECS Container Agent"
+
+  if [ -e /etc/ecs/ecs.config ] && ! grep -q "^\\s*ECS_LOGLEVEL=debug" /etc/ecs/ecs.config; then
+    info "Debug mode is already disabled."
+  
+  else
+    case "${init_type}" in
+    systemd)
+      if [ ! -d /etc/ecs ]; then
+        mkdir /etc/ecs
+      fi
+
+      sed -i '/^\s*ECS_LOGLEVEL=debug/d' /etc/ecs/ecs.config
```

Comment:
this sets the log level to debug, instead of unsetting it or setting it back to info